tofhir-server/src/main/scala/io/tofhir/server/model/JobMetadata.scala

```diff
@@ -0,0 +1,8 @@
+package io.tofhir.server.model
```

Comment:
We might not need Metadata classes at all if we find some other way to obtain implementation-specific metadata